linchpin.py

```diff
@@ -28,13 +100,81 @@ def copy_files(path, dir_list, config):
             src_file = src+filename
             shutil.copy(src_file, dest)
 
+def checkpaths():
+    """ checks whether the linchpin layout already exists in cwd"""
+    cur_dir = os.getcwd()
+    print os.listdir(cur_dir)
+    layout_files = ['layouts', 'topologies', 'linchfile.lpf']
+    for f in layout_files:
+        if f in os.listdir(cur_dir):
+            return True
+
+def parse_yaml(lpf):
+    """ parses yaml file into json object """
+    with open(lpf, 'r') as stream:
+        try:
+            lpf = yaml.load(stream)
+            return lpf
+        except yaml.YAMLError as exc:
+            print(exc)
+
+def invoke_linchpin(config, e_vars, playbook="PROVISION", console=True):
+    """ Invokes linchpin playbook """
+    playbook_path = config.clipath+"/provision/"+PLAYBOOKS[playbook]
+    inventory = Inventory(loader=config.loader, variable_manager=config.variable_manager,  host_list=[])
+    config.variable_manager.extra_vars = e_vars 
+    passwords = {}
+    utils.VERBOSITY = 4
+    pbex = PlaybookExecutor(playbooks=[playbook_path], inventory=inventory, variable_manager=config.variable_manager, loader=config.loader, options=config.options, passwords=passwords)
```

Comment:
Will be changing the script according to PEP8.
Here on before PR is raised, will run through PEP8 checks